.github/workflows/build.yml

```diff
@@ -82,6 +83,12 @@ jobs:
           key: ${{ runner.os }}-yarn-${{ hashFiles('**/yarn.lock') }}
           restore-keys: |
             ${{ runner.os }}-yarn-
+      - name: Install Hasura Cli
+        run: curl -L https://github.com/hasura/graphql-engine/raw/master/cli/get.sh | bash
```

Comment:
Do we want to specify a specific Hasura CLI version?

Reply:
I wouldn't since this is only used in CI and will help us find breaking changes in new versions if present.